### DISCUSSION TITLE
The Basic syntax documentation does not match with actual output

### DISCUSSION THREAD

**Discussion by prithvipal** [open-policy-agent/community#586] [Community]
I am new to OPA and I am learning Rego from the official document.
> If we evaluate v, the result is undefined because the body of the rule never evaluates to true. As a result, the document generated by the rule is not defined.
>
But when I run the following statement in interactive shell:
```
v if "hello" == "world"
```
Then I am getting error `1 error occurred: 1:1: rego_unsafe_var_error: var v is unsafe` where the document say it will show as `undefined decision`.

Is this document up to date?

**Comment by anderseknert** [comment]
Good morning! Using `if` (and other keywords, like `contains`) requires first having done an `import rego.v1`:

```
❯ opa run
OPA 0.64.1

Run 'help' to see a list of commands and check for updates.

> v if "hello" == "world"
1 error occurred: 1:1: rego_unsafe_var_error: var v is unsafe
> import rego.v1
> v if "hello" == "world"
Rule 'v' defined in package repl. Type 'show' to see rules.
> v
undefined
```

This is explained later down in the docs at https://www.openpolicyagent.org/docs/latest/policy-language/#rules
But perhaps that section should be moved up to before `if` is used in an example.
